.operations/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contribution model
+
+### Steering committee 🏆
+
+The steering committee work together to provide guidance and future direction to the project. Each committee member has a particular experitse which they share their knowledge on, and work to lead further improvements to the project in that are. In addition, each member of the committee leads a project which are tracked under Github projects.
+
+### Collaborators 👍
+
+Collaborators are members who are contributing to the repository on a reguar basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. They help the steering committee ensure that the content provided is of a high standard, up to date with the industry, and available in many languages. Members who are displaying the above activity, and quality of contribution will be invited to become a collaborator.
+
+On the occasion where a collaborator becomes inactive for some time, they will be asked if they would still lik, however will always be acknowledged as a past collaborator.
```

Comment:
```suggestion
On the occasion where a collaborator becomes inactive for some time, they will be asked if they would still like to contribute as a collaborator or leave their role, being acknowledged as a past collaborator.
```

Reply:
The change suggestion above is just that, a suggestion. I'm not completely sure how to rephrase this sentence, maybe we could find a common solution 👍

Reply:
Whoops, looks like I missed finishing that sentence